### DISCUSSION TITLE
Google custom SAML app integration with Keycloak

### DISCUSSION THREAD

**Discussion by ilyai** [keycloak/keycloak#14284] [Q&A]
Hello,

I'm trying to setup IdP initiated SSO with a Google Workspace app to work with Keycloak. Here's the description of the problem in details: https://stackoverflow.com/questions/73568876/google-custom-saml-app-integration-with-keycloak

I've tried contacting Google support but they didn't find any problems on their side.
I'm wondering if Keycloak 17 actually supports this?

Also this issue may be related: https://stackoverflow.com/questions/71545359/idp-initiated-sso-from-google-saml-with-keycloak-as-identity-broker

**Comment by anupmiet** [comment]
I am also facing similar issue in keycloak latest version. Whenever i click google saml aap icon, keycloak redirect URL is called up through google endpoint but cookie are not there causing sso fail.
